go/vt/mysqlctl/metadata_tables.go

```diff
@@ -88,34 +88,32 @@ func PopulateMetadataTables(mysqld MysqlDaemon, localMetadata map[string]string,
 	}
 	for _, sql := range sqlAlterLocalMetadataTable {
 		if _, err := conn.ExecuteFetch(sql, 0, false); err != nil {
-			if merr, ok := err.(*mysql.SQLError); ok && merr.Num == mysql.ERDupFieldName {
-				log.Errorf("Expected error executing %v: %v", sql, err)
-			} else {
+			// Ignore "Duplicate column name 'db_name'" errors which can happen on every restart.
+			if merr, ok := err.(*mysql.SQLError); !ok || merr.Num != mysql.ERDupFieldName {
 				log.Errorf("Unexpected error executing %v: %v", sql, err)
 				return err
 			}
 		}
 	}
 	sql := fmt.Sprintf(sqlUpdateLocalMetadataTable, dbName)
 	if _, err := conn.ExecuteFetch(sql, 0, false); err != nil {
-		log.Errorf("unexpected error executing %v: %v, continuing. Please check the data in _vt.local_metadata and take corrective action", sql, err)
+		log.Errorf("Unexpected error executing %v: %v, continuing. Please check the data in _vt.local_metadata and take corrective action.", sql, err)
```

Comment:
I think we should just remove 'Unexpected' from these messages (including the one above).

Reply:
Done. I squashed the commits since the changes are so trivial.